brave/brave-core
# Refactor Brave Ads unit tests/mocks/utils

Resolves https://github.com/brave/brave-browser/issues/9471

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Confirm ads are delivered as expected
- Confirm ad conversions work as expected

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
